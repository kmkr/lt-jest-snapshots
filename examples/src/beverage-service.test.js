import beverageService from './beverage-service.js'

describe('HotBeverage', () => {
    it('should render correctly', () => {
        const coffee = beverageService({
            type: 'coffee',
            centiliters: 50,
        });
        expect(coffee).toMatchSnapshot();        
    });

    it('should throw on too much content', () => {
        debugger;
        expect(() => {
            beverageService({
                centiliters: 150
            });
        }).toThrowError('Too much content');
    });

    it('should throw on too much content', () => {
        expect(() => {
            beverageService({
                centiliters: 150
            });
        }).toThrowErrorMatchingSnapshot();
    });
});