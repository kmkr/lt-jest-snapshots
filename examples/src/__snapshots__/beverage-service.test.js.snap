// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HotBeverage should render correctly 1`] = `
Object {
  "isHot": true,
  "sizeOfCup": "large-cup",
  "type": "coffee",
}
`;

exports[`HotBeverage should throw on too much content 1`] = `"Too much content"`;
